### PR TITLE
Fixed graphics runtime dtor order

### PR DESCRIPTION
**Pull request by PENGUINLIONG**

### CODE REVIEW

**Comment by PENGUINLIONG** [comment]
@jim19930609 does it actually work?

**Comment by jim19930609** [comment]
> @jim19930609 does it actually work?

Hmmm, actually no. I'm working on another fix that refactors the global variable `F` into a local variable (So the ti::aot_demo::Framework destructs before dlclose("libtaichi_c_api.so")), let me try again with these two PRs come together.
